code/modules/mob/living/simple_animal/abnormality/teth/mhz.dm

```diff
@@ -0,0 +1,79 @@
+/mob/living/simple_animal/hostile/abnormality/mhz
+	name = "1.76 MHz"
+	desc = "You can't see anything but static."
+	icon = 'ModularTegustation/Teguicons/96x96.dmi'
+	icon_state = "mhz"
+	icon_living = "mhz"
+	pixel_x = -32
+	base_pixel_x = -32
+	pixel_y = -32
+	base_pixel_y = -32
+	maxHealth = 400
+	health = 400
+	start_qliphoth = 4
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 40,
+		ABNORMALITY_WORK_INSIGHT = list(40, 30, 20, 20, 20),
+		ABNORMALITY_WORK_ATTACHMENT = list(20, 10, 0, 0, 0),
+		ABNORMALITY_WORK_REPRESSION = list(55, 55, 60, 60, 60)
+		)
+	work_damage_amount = 5
+	work_damage_type = WHITE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/noise,
+		/datum/ego_datum/armor/noise
+		)
+
+/mob/living/simple_animal/hostile/abnormality/mhz/work_chance(mob/living/carbon/human/user, chance)
+	if(get_attribute_level(user, FORTITUDE_ATTRIBUTE) < 40)
+		var/newchance = chance*1.25
+		return newchance
+	return chance
+
+/mob/living/simple_animal/hostile/abnormality/mhz/success_effect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(1)
+	if(pe >= datum_reference.max_boxes)
+		datum_reference.qliphoth_change(1)
+
+/mob/living/simple_animal/hostile/abnormality/mhz/neutral_effect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(40))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mhz/failure_effect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(80))
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mhz/work_complete(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(user.sanity_lost)
+		datum_reference.qliphoth_change(-1)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/mhz/zero_qliphoth(mob/living/carbon/human/user)
+	SSweather.run_weather(/datum/weather/mhz)
+	datum_reference.qliphoth_change(4)
+
+//We're gonna make it a weather that affects all hallways.
+/datum/weather/mhz
+	name = "static"
+	immunity_type = "static"
+	desc = "Static created by 1.76 MHz."
+	telegraph_message = "<span class='warning'>You feel ominous pressure.</span>"
+	telegraph_duration = 300
+	weather_message = "<span class='userdanger'><i>Are.... those the sounds of humans wailing? Are they suffering?</i></span>"
+	weather_overlay = "mhz"
+	weather_duration_lower = 1200		//2-3 minutes.
+	weather_duration_upper = 1800
+	end_duration = 100
+	end_message = "<span class='boldannounce'>It's all calm once more. You feel at peace.</span>"
+	area_type = /area/facility_hallway
+	target_trait = ZTRAIT_STATION
+
+/datum/weather/mhz/weather_act(mob/living/carbon/human/L)
+	if(ishuman(L))
+		L.apply_damage(9, WHITE_DAMAGE, null, L.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
```

Comment:
I just took what was in the game and doubled it as per usual. Also, main rooms are safe. 
I can reduce to 6

Reply:
Also keep in mind that it may be mutually exclusive with Staining Rose, as the two of them together are simply too fucking brutal.
I can just remove the weather restrictions of stopping one and starting another but I feel that an effect that hits the whole station in the same way these two do shouldn't stack in that way

Reply:
No, I think it'd be better if it spread like space vines instead.
Weather effects aren't that good.

Reply:
Ill honestly just cut it from this PR and deal with it later. Blue Shepard is the focus, 1.76 MHz is an afterthought.